src/pages/learn/execution.mdx

```diff
@@ -114,39 +120,37 @@ Human: {
 }
 ```
 
-Notice that our type system claims `appearsIn` will return Enum values with known values, however this function is returning numbers! Indeed if we look up at the result we'll see that the appropriate Enum values are being returned. What's going on?
+Notice that our type system claims `appearsIn` will return Enum types with known values, however, this function is returning numbers! Indeed if we look up at the result we'll see that the appropriate values for the Enum type are being returned. What's going on?
 
-This is an example of scalar coercion. The type system knows what to expect and will convert the values returned by a resolver function into something that upholds the API contract. In this case, there may be an Enum defined on our server which uses numbers like `4`, `5`, and `6` internally, but represents them as Enum values in the GraphQL type system.
+This is an example of _scalar coercion_. The type system knows what to expect and will convert the values returned by a resolver function into something that upholds the API contract. In this case, there may be an Enum type defined on our server that uses numbers like `4`, `5`, and `6` internally, but represents them as the expected values in the GraphQL type system.
```

Comment:
Hi @mandiwise @benjie, this section is called Scalar coercion but an Enum is used as example. Since these are two separate types, conflating the two could lead to confusion for the readers.

I believe the spec uses the term `Result coercion` to describe the concept in this section. Maybe the following minor changes could make it clearer:

- Changing `scalar coercion` to `result coercion`
- Adding an example of the enum coercion implementation e.g.

```ts
Episode: {
  NEWHOPE: 4
  EMPIRE: 5
  JEDI: 6 
}
```

Sorry for the late comment, I just happen to see this PR after it's merged. I'm happy to make contributions if the above makes sense.

Reply:
I agree the mix of scalar/enum is confusing and would love to see your suggested edits in a PR 👍